transforms/language/language_id/python/Makefile

```diff
@@ -0,0 +1,56 @@
+# Define the root of the local git clone for the common rules to be able 
+# know where they are running from.
+REPOROOT=../../../..
+# Include a library of common .transform.* targets which most
+# transforms should be able to reuse.  However, feel free
+# to override/redefine the rules below. 
+
+# $(REPOROOT)/.make.versions file contains the versions
+
+TRANSFORM_RUNTIME=python
+TRANSFORM_NAME=lang_id
+DOCKER_IMAGE_VERSION=${LANG_ID_VERSION}
```

Comment:
you may want to change this to LANG_ID_PYTHON_VERSION (in .make.versions at the top of the repo).